docs/deployment.md

```diff
@@ -1,66 +1,182 @@
 # Deployment
 
-One of the key features of BigFlow CLI is the full automation of the build and deployment process.
-BigFlow can [build](project_setup_and_build.md) your workflows
-to dockerized Airflow DAGs and deploy them to Google Cloud Composer.
+One of the key features of BigFlow is the full automation of the build and deployment process.
+BigFlow can dockerize your workflows and deploy them to Google Cloud Composer.
 
-## Deployment artifacts
-There are two deployment artifacts:
+## GCP runtime environment
+
+BigFlow GCP runtime environment consists of two services:
+
+1. Google [Cloud Composer](#cloud-composer),
```

Comment:
Dlaczego "Google" jest poza nawiasem? Dziwnie to wygląda.

![Zrzut ekranu 2020-09-24 o 10 34 59](https://user-images.githubusercontent.com/9019407/94121557-bf174200-fe51-11ea-9347-9aad74c9f9e9.png)